src/style.css

```diff
@@ -1,3 +1,75 @@
+:root {
+  --link-color: #00acdc;
```

Comment:
there are black and blue links on the site, that is, the name of the variable does not say that it is just the color of the link this color is used in the logo and for the decorative line under the link, which means it is an accent color